src/main/webapp/assets/modules/employer-dashboard/employerDashboardController.js

```diff
@@ -1,4 +1,4 @@
-techlooper.controller('employerDashboardController', function ($scope, jsonValue, utils, apiService, $location, $filter) {
+techlooper.controller('employerDashboardController', function ($scope, jsonValue, utils, apiService, $location, $filter, $rootScope) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'$rootScope' is defined but never used.